vm-setup/roles/v1aX_integration_test/tasks/remediation.yml

```diff
@@ -89,23 +106,25 @@
   - name: Power cycle a single worker node
     include: power_cycle.yml
     vars:
-      NODE: '{{ WORKER_NODE }}'
+      NODE: '{{ WORKER_BMH }}'
```

Comment:
I would rather call this variable BMH_NAME, NODE is cryptic

Reply:
Done .